kymatio/frontend/sklearn_frontend.py

```diff
@@ -0,0 +1,41 @@
+from sklearn.base import BaseEstimator, TransformerMixin
+
+
+class ScatteringTransformerMixin(BaseEstimator, TransformerMixin):
+    def fit(self, x=None, y=None):
+        # No fitting necessary.
+        return self
+
+    def predict(self, x):
+        n_samples = x.shape[0]
+
+        x = x.reshape((-1,) + self.shape)
+
```

Comment:
space?

Reply:
Between `,` and `)`? I don't think that's standard. Seems PEP8 [compliant](https://pep8.org/#when-to-use-trailing-commas).

Reply:
the line 13 and 11, are they needed?

Reply:
Ah. Not really. Again, just following what came before here. Will remove.